docs/environment-variables.md

```diff
@@ -19,3 +19,32 @@ The OAuth access token, generated by Launchpad.
 ### LP\_API\_TOKEN\_SECRET
 
 The OAuth token secret, generated by Launchpad.
+
+## GitHub settings
+
+### GITHUB_API_ENDPOINT
+- **Example**: `http://localhost:4000/github`, `https://api.github.com`
+
+The base URL for the GitHub API.
+
+### GITHUB_USERNAME
+- **Example:** katiefenn
+- **DEPRECATED**
+
+The GitHub account name to use for authenticating with the GitHub API.
+**WARNING: Do not commit GitHub credentials to the repository. Keep them secure on your computer.**
+**WARNING: This setting is deprecated and subject to removal when Oauth authentication is implemented**
+
+### GITHUB_PASSWORD
+- **Example:** password
+- **DEPRECATED**
+
+The GitHub account password to use for authenticating with the GitHub API.
+**WARNING: Do not commit GitHub credentials to the repository. Keep them secure on your computer.**
+**WARNING: This setting is deprecated and subject to removal when Oauth authentication is implemented**
```

Comment:
GitHub HTTP authentication is indended as a stop-gap until Oauth is implemented in a later PR.

I recommend copying `development.env` to a file outside of your local repository and adding settings for `GITHUB_USERNAME` and `GITHUB_PASSWORD` if you absolutely must test against the production GitHub API.

Alternatively, the mock service will work with any username and password.